root/statistics/MusicBrainzHistory.js

```diff
@@ -27,13 +26,13 @@ component MusicBrainzHistory(events: $ReadOnlyArray<StatisticsEventT>) {
                   <a href={event.link}>{title}</a>
                 ) : title}
               </h2>
-              <p>{expand2react(event.description)}</p>
+              <p>{l_statistics(event.description)}</p>
               <hr />
             </div>
           );
         }) : (
           <p>
-            {l_statistics('It seems we have no history to show at all!')}
+            {l_history('It seems we have no history to show at all!')}
```

Comment:
This is neither history event title nor description. It should rather stay in the domain `statistics`.
(Same as in a previous comment)

Reply:
Hmm. It seems a bit weird to have this in `statistics` because it's explicitly about the history though? I'd be half-tempted to move it to `server` instead of `statistics` if you think `history` is a bad fit.

Reply:
Oh I see, this is the page `/history` that can be accessed from the menu “MusicBrainz History”. It should probably be moved to `/root/history/` then.

Reply:
In this case, moving to the domain `server` would make sense indeed.